californium-core/src/main/java/org/eclipse/californium/core/observe/InMemoryObservationStore.java

```diff
@@ -36,8 +44,33 @@
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(InMemoryObservationStore.class.getName());
 	private final ConcurrentMap<Token, Observation> map = new ConcurrentHashMap<>();
+	private volatile boolean enableStatus;
 
-	public InMemoryObservationStore() {
+	public InMemoryObservationStore(NetworkConfig config) {
+		
+		if (LOGGER.isInfoEnabled()) {
```

Comment:
INFO would be one of the adequate production log levels FMPOV. How do you prevent the excessive logging in this case? Maybe a flag in NetworkConfig for enabling/disabling this would be more appropriate than just guarding by log level?

Reply:
We may use HEALTH_STATUS_INTERVAL of 0 to disable it.

Reply:
Or use a different LOGGER?